cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -82,6 +105,26 @@ function log() {
   esac
 }
 
+# Generate kubectl prune-whitelist flags from provided resource list.
+function generate_prune_whitelist_flags() {
+  resources=($@)
+  for resource in ${resources[@]}
```

Comment:
quote variable expansions, i.e. prefer "${resources[@]}".

move the do to this line to be consistent with our bash style.

Reply:
Will do.

Reply:
Done.